MiaWrapper.py

```diff
@@ -1,18 +1,18 @@
 import argparse
 import json
 import os.path
-
 import PyradiomicsWrapper
-
+from shutil import copyfile
 
 def miaWrapper(computationDataJson, paramsFile, outputPath):
-    
     with open(computationDataJson) as computationDataJsonFile:
         computationDataJsonObject = json.load(computationDataJsonFile)
     
     images = computationDataJsonObject['images']
-    images = sortImages(images, ['CT', 'RTSTRUCT'])
-    
+    modalities = ['CT', 'RTSTRUCT']
+    if checkIfCopyIsNeeded(images, modalities):
```

Comment:
Also some short comment on why we feel this is a temporary workaround to a bigger problem.